collections/_infographics/emissions/emission-reduction-potential.md

```diff
@@ -3,12 +3,139 @@ layout:        infographic
+
+
+For shorter distances within Europe, planes can be partially replaced by train services, but for long flights, it is difficult to imagine a substitute. Technologies that would allow air travel with significantly lower emissions are still being tested. However, air transport is currently in crisis due to the COVID-19 epidemic and it is possible that the perception of its importance will change in the future.
+
+### We consume half the amount of dairy / meat
+
+According to the Czech Statistical Office, the average Czech eats about 70 kg of meat a year, of which about 6 kg is beef. When all types of meat are included, including the distinction between dairy and beef cattle, the emissions related to meat consumption in the Czech Republic total 6.8 million tonnes of <glossary id="co2eq">CO<sub>2</sub>eq</glossary>. This figure includes the entire production cycle, i.e. it includes, for example, feed and transport to the market. At the same time, we should note that part of the meat production is imported into the Czech Republic. So the value corresponds to the emissions related to the meat consumed, not the meat produced in the Czech Republic. **If people consumed half the amount of meat, 3.4 million tonnes of CO<sub>2</sub>eq emissions would be saved annually.**
+
+Together, the consumption of milk and milk products can be expressed in terms of the value needed for production. The average consumption is 239 litres of milk per person per year (equivalent to 245 kg). The entire cycle of producing one kg of milk (including feed, methane production, refrigeration, transport, etc.) produces an average of 2.8 kg CO<sub>2</sub>eq of greenhouse gas emissions. The total emissions associated with the consumption of milk and dairy products in the Czech Republic are 7.3 million tonnes CO<sub>2</sub>eq. If the Czech Republic's population halved its dairy product consumption, 3.7 million tonnes of CO<sub>2</sub>eq would be saved.
+
+Many studies suggest that changes in dietary habits could positively impact health, for example, because approximately 71% of men and 57% of women in the <!--[-->Czech Republic are overweight<!--](https://www.ceskovdatech.cz/clanek/85-povolujeme-opasky-cechu-s-nadvahou-vyrazne-pribyva/)-->. Whether changing dietary habits is realistic for a large part of the population is a question beyond the scope of our text.
```

Comment:
Unable to find this link in English. Not sure if there is an equivalent to https://www.ceskovdatech.cz/clanek/85-povolujeme-opasky-cechu-s-nadvahou-vyrazne-pribyva